### ISSUE TITLE
[pathtracer.cpp] 'subRegion' was not defined in this scope.

### ISSUE THREAD

**Issue by bartoszek** [LuxCoreRender/LuxCore#240]
After 0c7052a546e2a3a31d3dcf532921735f304a31e8 I'm no longer able build on ArchLinux.
```
...
/build/luxcorerender-git/src/LuxCore/src/slg/engines/pathtracer.cpp: In member function ‘void slg::PathTracer::ConnectToEye(u_int, luxrays::IntersectionDevice*, const slg::Scene*, const slg::Film*, float, float, float, float, const slg::LightSource&, const slg::BSDF&, const Spectrum&, const slg::LightPathInfo&, std::vector<slg::SampleResult>&) const’:
/build/luxcorerender-git/src/LuxCore/src/slg/engines/pathtracer.cpp:699:36: error: ‘subRegion’ was not declared in this scope
  699 |     assert (sampleResult.pixelX >= subRegion[0]);
      |                                    ^~~~~~~~~
make[2]: *** [src/slg/CMakeFiles/slg-core.dir/build.make:271: src/slg/CMakeFiles/slg-core.dir/engines/pathtracer.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:255: src/slg/CMakeFiles/slg-core.dir/all] Error 2
make: *** [Makefile:84: all] Error 2
```
SysInfo: { gcc=9.1.0, llvm=8.0.1, boost= 1.69 }
[Build log](https://github.com/LuxCoreRender/LuxCore/files/3693354/make.log)
[Travis log](https://travis-ci.org/bartoszek/AUR-luxcorerender-git/builds/593627990)

**Comment by Dade916** [comment]
You are compiling with assertion enabled (not sure if it is intended). Anyway I should have fixed the problem.

**Comment by bartoszek** [comment]
Solved in 6fbe988ebdf188783b660a2c7a7a82afee55e592
